inbm/dispatcher-agent/dispatcher/sota/snapshot.py

```diff
@@ -157,6 +163,26 @@ def take_snapshot(self) -> None:
             self._dispatcher_callbacks.broker_core.telemetry("SOTA snapshot succeeded")
             self.snap_num = snapshot_num
 
+    def delete_snapshot(self) -> int:
+        """Deletes snapshots from the directory"""
+        # Check if the specified path is a directory
+        if os.path.isdir(SNAPSHOT_DIRECTORY_PATH):
+            # Get a list of files in the directory
+            # delete
+            filename = 'rootConfig' 
+            rc, err = self.trtl.delete_config(SNAPSHOT_FILENAME)
```

Comment:
It looks like we're passing a file to trtl here when we want to pass a config name. Also, we can't just ask snapper to delete a config without listing the snapshots under this config and deleting them one by one.